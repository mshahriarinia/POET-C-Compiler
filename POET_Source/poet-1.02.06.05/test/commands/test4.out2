
DEBUGGING at level 1
LEN(A)
===>2
END DEBUGGING at level 1

DEBUGGING at level 1
HEAD(B)
===>4
TAIL(B)
===>((5 7) (9 10))
LEN(B)
===>3
END DEBUGGING at level 1

DEBUGGING at level 1
LEN(C)
===>2
END DEBUGGING at level 1
