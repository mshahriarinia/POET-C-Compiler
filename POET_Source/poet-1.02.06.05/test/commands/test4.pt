include utils.incl

<parameter A type=((INT,INT),INT) default=((1,2), 3) message="affinity relations"/> 
<parameter B type=(INT|(INT...)...) parse=LIST(INT|IntList," ")  default=((1 2)  3) message="affinity relations"/> 
<parameter C type=( (INT,INT),INT) default=((1,2),3) parse=TUPLE( "" TUPLE("(" INT "," INT ")") "," INT "") />
<parameter D type=0..5 default=2 />
<parameter E type=0.._ default=80 />
<eval DEBUG{
    A;
    A[0];
    A[1];
    LEN(A);
}/>

<eval DEBUG{
   B;
   car B;
   cdr B;
   LEN(B);
}/>

<eval DEBUG{
    C;
    C[0];
    C[1];
    LEN(C);
}/>

