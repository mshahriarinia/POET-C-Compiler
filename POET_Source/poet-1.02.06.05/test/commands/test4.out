
DEBUGGING at level 1
LEN(A)
===>2
END DEBUGGING at level 1

DEBUGGING at level 1
HEAD(B)
===>(1 2)
TAIL(B)
===>3
LEN(B)
===>2
END DEBUGGING at level 1

DEBUGGING at level 1
LEN(C)
===>2
END DEBUGGING at level 1
